qtpy/tests/test_qtcore.py

```diff
@@ -46,36 +46,82 @@ def test_qtime_toPython():
     assert isinstance(py_time, time)
 
 
+def test_qdatetime_toPyDateTime():
+    """Test QDateTime.toPyDateTime"""
+    q_date = QtCore.QDateTime.currentDateTime()
+    assert QtCore.QDateTime.toPyDateTime is not None
+    py_date = q_date.toPyDateTime()
+    assert isinstance(py_date, datetime)
```

Comment:
Maybe worth passing a fixed datetime (and similar, for the others) and checking if the resulting object equals the expected?

Reply:
Makes sense. This way, we'll check that the conversion is valid.